src/main/java/edu/ucmo/spring_example/model/Truck.java

```diff
@@ -0,0 +1,87 @@
+package edu.ucmo.spring_example.model;
+
+import java.util.Objects;
+import javax.persistence.*;
+
+@Entity
+@Table(name = "truck")
+public class Truck {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private int id;
+    @Column
+    private String make;
+    @Column
+    private String model;
+    @Column
+    private int year;
+    @Column
+    private int type;
```

Comment:
I think this should be an enum instead of an int.

Reply:
No, it's an int.

Reply:
Ooops I thought about it it should be an enum.

Reply:
Looks good.